### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KomashiFX**
more beautiful to eyes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,37 @@
...
-
-## Features:
-* Displaying off:
-   + current map, as well as closes point of interest
-   + current raid or fractal boss
-   + character name, race and profession (elite spec)
-   + commander icon if player is currently commanding a squad
-   + active guild tag (needs API key)
-   + time spent on map
+<p align="center"><img alt="Guild Wars 2 RPC" src="https://media.discordapp.net/attachments/908219512271286282/965798895433170954/GW2.png?width=473&height=473" height="76"></p></img>
```
Comment (n1tr0-5urf3r) [inline README.md:1]: Allthough I really like your new design of the logo, I'd prefer to have the "official" logo there. You can find it hosted on the gw2rpc Server here: https://gw2rpc.info/static/img/logo.png

We could probably create a new logo for future versions if you are skilled :)

File: README.md [hunk 2]
```diff
@@ -1,22 +1,37 @@
...
+
+<h1 align="center">Guild Wars 2 RPC</h1>
+<p align="center">A Discord Rich Presence addon for Guild Wars 2.</p>
+
+<a href="https://gw2rpc.info"><p align="center">Go to website</a>・<a href="https://github.com/Maselkov/GW2RPC/releases">Download Latest</a>・<a href="https://github.com/Maselkov?tab=repositories">Other Projects...</p></a>
+
+---
+<p align="center">You need install <img src="https://api.iconify.design/bi:discord.svg?color=%23f9f9f9" height="12"> <b><a href="https://discord.com/download">Discord Desktop version</b></a> to run the rich presence.</p>
```
Comment (n1tr0-5urf3r) [inline README.md:9]: Small grammar: "You need **to** install"

File: README.md [hunk 3]
```diff
@@ -1,22 +1,37 @@
...
 * Automatic raid announcer
 * Automatic update checking
 * Web based registry for maps
 * Configurable settings
 * Supports multiple accounts
----
-## How to install
+</details>
+
+<details markdown='1'><summary>To next features</summary>
```
Comment (n1tr0-5urf3r) [inline README.md:28]: I'd rather not have a rickroll in the official readme. If it helps you, you got me :P

File: README.md [hunk 4]
```diff
@@ -1,22 +1,37 @@
...
+<p align="center">A Discord Rich Presence addon for Guild Wars 2.</p>
+
+<a href="https://gw2rpc.info"><p align="center">Go to website</a>・<a href="https://github.com/Maselkov/GW2RPC/releases">Download Latest</a>・<a href="https://github.com/Maselkov?tab=repositories">Other Projects...</p></a>
+
+---
+<p align="center">You need install <img src="https://api.iconify.design/bi:discord.svg?color=%23f9f9f9" height="12"> <b><a href="https://discord.com/download">Discord Desktop version</b></a> to run the rich presence.</p>
+
+<h2><img src="https://api.iconify.design/ic:baseline-auto-awesome.svg?color=%23ff8cf3" height="20">・Features</h2>
+
+<details markdown='1'><summary>Displaying off</summary>
```
Comment (n1tr0-5urf3r) [inline README.md:13]: Is it possible to unfold the features menu by default?

**Review by n1tr0-5urf3r — CHANGES_REQUESTED** [review]
Looks nice, got some open points, ty for the PR :)

File: README.md (hunk 2 above)
Comment (KomashiFX) [reply README.md:9]: sorry my english is bad :b

File: README.md (hunk 4 above)
Comment (KomashiFX) [reply README.md]: I was trying to find a way to improve this, but I couldn't, so I put it the same way it's in the official readme

**Pull request merged**